CMakeLists.txt

```diff
@@ -40,17 +58,40 @@ set(NOVELRT_VULKAN_VERSION "1.3.211" CACHE STRING "Vulkan version")
 set(NOVELRT_VULKAN_MEMORY_ALLOCATOR_VERSION "2.3.0" CACHE STRING "VulkanMemoryAllocator version")
 set(NOVELRT_VULKAN_SDK_PATH "" CACHE STRING "Directory for NovelRT to find the Vulkan SDK (mainly for macOS support)")
 
-find_package(Doxygen ${NOVELRT_DOXYGEN_VERSION}
-  COMPONENTS dot
-)
+if(NOVELRT_PIE)
+  set(CMAKE_POSITION_INDEPENDENT_CODE TRUE)
```

Comment:
We should use the POSITION_INDEPENDENT_CODE property instead of a global CMake variable: https://cmake.org/cmake/help/latest/prop_tgt/POSITION_INDEPENDENT_CODE.html